tests/benchmarks/cuda/benchmark_propagator_cuda.cpp

```diff
@@ -45,6 +45,7 @@ void fill_tracks(vecmem::vector<free_track_parameters<transform3>> &tracks,
 
 template <propagate_option opt>
 static void BM_PROPAGATOR_CPU(benchmark::State &state) {
+    toy_cfg.do_check(false);
```

Comment:
Does this not work, if you add it in the .cpp in ln. 32?

Reply:
no. it doesn't compile

Reply:
Odd, it compiles for me